README.md

```diff
@@ -95,6 +95,13 @@ To see all supported options and arguments, run:
 
     cargo run -- --help
 
+## Testing
+
+The test suite includes end-to-end tests.
+They share sockets, and should be run in a serial manner:
```

Comment:
Kudos for adding this to README :+1:  but please don't join the "break the line for just starting a new sentence randomly" crowd. :)

```suggestion
The test suite includes end-to-end tests. They share sockets, and should be run in a serial manner:
```